yarn.lock

```diff
@@ -2449,6 +2449,19 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@hypothesis/annotation-ui@npm:^0.1.1":
+  version: 0.1.1
+  resolution: "@hypothesis/annotation-ui@npm:0.1.1"
+  peerDependencies:
+    "@hypothesis/frontend-shared": ^9.4.0
+    dompurify: ^3.0.0
+    escape-html: ^1.0.3
```

Comment:
Only @hypothesis/frontend-shared really needs to be a peer dependency here. None of the others are used by the client outside of annotation UI rendering.